README.md

```diff
@@ -381,6 +381,9 @@ Converts ISIN to IEX Cloud symbols.
 ```ruby
 symbols = client.ref_data_isin(['US0378331005'])
 
+# You can also pass single elements as string:
```

Comment:
I am going to nitpick. It's not clear that you can pass an array of ISINs here because the sample has an array of 1. 

I think you want both examples and not a comment.

Convert ISIN to IEX Cloud symbols.

example with 1

The API also lets you convert multiple ISINs to IEX Cloud symbols.

example with many